h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -661,29 +661,34 @@ public static void main( String[] args ) {
         Chunk c0 = chks[cols                 ];
         Chunk c1 = chks[cols+(chks.length>>1)];
         for( int rows = 0; rows < chks[0]._len; rows++ ) {
-          if (c0 instanceof C16Chunk && c1 instanceof C16Chunk) {
```

Comment:
you also need to cover situation when one is NA and the other is not.

It should be something like if(c0.isNA(i) != c0.isNa(i)) _uneaquel = true; break;
if(!c0.isNA(i)){....